CHANGELOG.md

```diff
@@ -22,6 +22,8 @@
 - Update `buf format -w` so that it does not touch files whose contents don't actually
   change. This eliminates noisy notifications to file-system-watcher tools that are
   watching the directory that contains proto sources.
+- Fixes issue where `buf breaking` or `buf lint` might fail when a `.proto` file contains
+  a message or field definition nested deeply in other message definitions.
```

Comment:
I'm not entirely convinced we should have a changelog entry for this, since it doesn't really give users a clear indication of what changed. I think I would remove the changelog entry entirely from this PR.